apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -954,6 +957,13 @@ type ResourcePlacementStatus struct {
 	// +kubebuilder:validation:MaxItems=100
 	DiffedPlacements []DiffedResourcePlacement `json:"diffedPlacements,omitempty"`
 
+	// ObservedResourceIndex is the index of the resource snapshot that is currently being rolled out to the given cluster.
+	// During rollout, depending on the rollout strategy, clusters may observe different resource indices.
```

Comment:
what does this mean?

Reply:
For example, with concurrent updateRuns, different clusters in different stages can have different resource snapshot indices, one with version 1 and one with version 2 for example.